sw/device/lib/dif/README.md

```diff
@@ -12,3 +12,54 @@ libraries.
 There is one DIF library per hardware IP, and each one contains the DIFs
```

Comment:
Could you please for the first occurence of acronym `DIF` use full name. Like `Device Interface Functions (DIF)`.

Reply:
There is one at the top of the file.